website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,229 @@
+
+#### Business needs
+
+<!-- TODO -->
+
+Organizing fields based on their importance (high to low) to successful completion of a form can help minimize abandonment; a crucial aspect to meeting business goals and metrics.
+
+
+#### Logical grouping
+
+Fields should be ordered "logically"; consider how a user will fill in information based on the context they are in.
+
+- When filling out a payment form: organize and group the fields in the same order as they appear on a credit card or payment method.
+  - Name, card number, expiration date, security code
+<!-- TODO: add more examples -->
+
+#### Visual organization
+
+Once a logical organization or grouping has been established,
+
+- categorize elements and fields into sections, 
+- introduce typographic elements to establish hierarchy,
+- and if necessary use dividers to more clearly differentiate sections.
+
+Each one of these methods will visually help the user to better parse and understand the relationships between each section and fields contained within.
+
+## Sizing
+
+### Length
+
+Determined by the number of fields and supporting elements (text, titles, etc). form length can have a significant impact on whether a user successfully completes a form or abandons the task prior to completion. If a feature or flow relies on a large number of fields, consider breaking up the form using one or more of the following strategies.
+
+#### Multiple sections
+
+Breaking up a form into multiple sections based on the relationship between fields and categories of sections can make a long form seem less complex. Introducing typographic elements and dividers can further aid the hierarchy of the form and differentiate sections from one-another.
+
+#### Multiple steps or pages
+
+For exceedingly long, complex forms (e.g., creating a cluster), consider breaking the form into multiple steps or pages. A [Stepper](/components/stepper-indicator) can be used to annotate status and location in a multi-step form.
+
+!!! Info
+
+As a general note, special care should be taken to _reduce_ the number of fields whenever possible.
+
+- Long forms are more challenging for the user which can result in a lower completion rate.
+- If a field is optional, consider whether it should be included in the form at all. If the information isn't crucial to the experience or feature, why is it being collected at all?
+!!!
+
+#### Using dividers
+
+Dividers introduce more visual hierarchy and differentiation in long, complex forms.
+
+Only use dividers between sections of a form, not between fields or other smaller elements within a section. A divider should be used to break up different types of content and categories within a form.
+
+When using a divider, increase the spacing between sections to 48px from 32px (24px on the top and bottom of the divider). This helps to separate the divider from surrounding form elements, otherwise it can appear "attached" to a specific section or field.
```

Comment:
Instead of saying "When using a divider, increase the spacing between sections" (which can be confusing) we can just say that the divider should have `24px` of margin above/below (it's the same thing, but it's simpler as a concept to understand)